app/ota_client.py

```diff
@@ -897,18 +898,20 @@ def _process_regular_files(self, rootfs_dir, rfiles_list, target_dir):
                 "staging-dict-_rollback_dict": manager.dict(),
                 "staging-_kernel_files": manager.dict(),
             }
+            await_c = manager.list()
```

Comment:
`await` reminds me of python3 async/await.
I prefer `pid_list` or some such name.

Reply:
But actually this variable works as await though, it waits until all processes finish their work, so I think this naming is meaningful?

Reply:
umm, for me, this variable is List. Waiting list asynchronously is code, not variable.
https://github.com/tier4/ota-client/pull/42/files#diff-4e43d11537bf0064fc07412772bff9e85eae2bc9334ee70950cd7217088600f0R947